### DISCUSSION TITLE
Polymorphic work around

### DISCUSSION THREAD

**Discussion by icemancast** [statamic-rad-pack/runway#318] [Feature Requests]
Feature request for polymorphic support. I have a need for images that is polymorphic for a resource. I tried the hasmany field but that broke the control panel.

**Comment by duncanmcclean** [comment]
I've already opened a [feature request for adding Polymoprphic relationship support](https://github.com/duncanmcclean/runway/discussions/245) to Runway. Unfortunately, it will take a lot of work to implement so probably won't be added anytime soon, unless someone opens a PR.
